### ISSUE TITLE
Regional travel times are not updated

### ISSUE THREAD

**Issue by nkuehnel** [msmobility/silo#72]
regional travel times in [Accessibility](https://github.com/msmobility/silo/blob/master/src/main/java/de/tum/bgu/msm/data/Accessibility.java) are currently not updated. When using MATSim travel times can change, meaning that zone to region travel times should be updated as well

**Comment by nkuehnel** [comment]
this has been solved in recent updates
